40-car-detector/README.md

```diff
@@ -0,0 +1,229 @@
+8. When a car enters the shot, click the Capture Data button or use the hotkey Ctrl+Shift+s. This will capture whatever's in the Frame Buffer and save it as a .jpg file in the car.class folder:
+    <p align="center">
+    <img src="assets/example_sample.jpg"/>
+    </p>
+    We recommend collecting data like this until you've got around 100 images. Using less data than that is likely to result in poor model performance.
+
+## Training the Model
+
+Now that you've got some data, it's time to train your model.
+
+1. [Sign up for an Edge Impulse account](https://studio.edgeimpulse.com/signup).
+2. Once you've signed in, click "Create new project", enter a project name, and click "Create new project".
+3. From here, follow Edge Impulse's ["Detect objects with centroids" tutorial](https://docs.edgeimpulse.com/docs/tutorials/detect-objects-using-fomo). This tutorial will guide you through labeling your data, specifying the deep learning model, training it, and testing its performance.
+
+If you're unhappy with the performance of the model, we have two suggestions:
+
+1. Collect more training data.
+2. Try out some of the "Expert mode tips" from [this Edge Impulse article](https://docs.edgeimpulse.com/docs/edge-impulse-studio/learning-blocks/object-detection/fomo-object-detection-for-constrained-devices#expert-mode-tips). Specifically, we found that increasing the `object_weight` parameter from 100 to 1000 was extremely helpful in getting better model performance.
+
+## Deploying the Model
+
+With your model trained, you'll now deploy it to the OpenMV cam's MCU.
+
+1. With your Edge Impulse project open, click Deployment in the left-hand toolbar.
+2. Search for "openmv" in the deployment options search bar and pick OpenMV Firmware.
+3. Click Build and wait for the download to complete.
+4. Follow [the "Deploying your impulse as an OpenMV firmware" section of this Edge Impulse article](https://docs.edgeimpulse.com/docs/deployment/running-your-impulse-openmv#deploying-your-impulse-as-an-openmv-firmware) to get the model onto your microcontroller using the OpenMV IDE.
+
+## Notehub Setup
+
+In order to push data from the Notecard to the cloud, you need to sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-pi/#set-up-notehub).
+
+## MicroPython Code
+
+The script that will run on the MCU is `main.py`. It depends on [note-python](https://github.com/blues/note-python), a Python library for communicating with a Notecard.
+
+### note-python
+
+To get the note-python files onto the MCU, use the `setup_board.py` script. First, you must identify the MCU's serial port. On Linux, it'll typically be something like `/dev/ttyACM0`. Once you have that, run `python setup_board.py <serial port>`, replacing `<serial port>` with your serial port. This script does a few things:
```

Comment:
> you must identify the MCU's serial port

Is there anything you could link to to help people with this? I would have no idea what to do.